src/lib/pickles/evaluation_lengths.ml

```diff
@@ -6,6 +6,7 @@ open Plonk_types
 let create (type a) ~(of_int : int -> a) : a Plonk_types.Evals.t =
   let one = of_int 1 in
   { w = Vector.init Columns.n ~f:(fun _ -> one)
+  ; coefficients = Vector.init Columns.n ~f:(fun _ -> one)
```

Comment:
I guess same comment here, it'd be good to introduce a different constant for coefficients